Automattic/simplenote-ios
# Display Published Icon for individual Notes

### Details:
- We're now displaying an Accessory ImageView whenever a note is published
- **SPTransitionController** has been heavily tuned so that animations don't flicker when showing / hiding the Shared indicator
- New tool in charge of rendering a Note into a UIImage (wooo!)
- Thank you @astralbodies !!!! This PR was... days in the making

Closes #312

### Testing:
1. Launch Simplenote
2. Open any note
3. Press the ( i ) indicator and **publish** the note
4. Go back to the Notes List
5. Verify that the (newly) shared note displays a new asset on the right

**Please:** Repeat this test in:

- [x] iPhone 8 Plus
- [x] iPhone Xs
- [x] iPhone Xs Max
- [x] iPad Air Fullscreen
- [x] iPad Air Sideover
- [x] iPad Air Multitasking

**Plus** make sure that this works with:
- [x] Notes that contain a single preview line (IE. note with just one line of text)
- [x] Notes with multiple preview lines
